packages/core/admin/admin/src/StrapiApp.js

```diff
@@ -33,36 +33,67 @@ const queryClient = new QueryClient({
   },
 });
 
-// FIXME
 const appLocales = Object.keys(translations);
 
 class StrapiApp {
   constructor({ appPlugins }) {
     this.appPlugins = appPlugins || {};
-    this.componentApi = Components();
-    this.fieldApi = Fields();
-    // FIXME
-    this.middlewares = [];
+    this.library = Library();
```

Comment:
You could eventually inject them through the constructor instead of coupling StrapiApp with the two classes. NITPICKY, but IoC is great 😊